source/guides/core.rst

```diff
@@ -14,8 +14,15 @@ Development Process
    /process/release-process*
    /process/accepting-pull-request*
    /process/community-systems*
-   /process/documentation-guidelines*
+   /process/documentation-guidelines.md
 
+Documentation Style Guide
+=========================
+
+.. toctree::
+   :maxdepth: 3
+
+   /process/sg_mattermost-doc-style.rst
```

Comment:
We discussed among PMs and propose we remove the `sg_` prefix from the style guide doc file names for now.

The `sg_` prefix will appear in the URL when viewing the page on docs.mattermost.com, and could be confusing if a user doesn't associate `sg_` with `style guide`.

We can consider file naming convention later.